test/contracts-network/colony-network-extensions.js

```diff
@@ -163,6 +163,36 @@ contract("Colony Network Extensions", (accounts) => {
       await checkErrorRevert(extension.install(colony.address), "ds-auth-unauthorized");
     });
 
+    it("allows a root user to install multiple extensions in a single transaction via multicall", async () => {
+      const installExtension1Data = await colony.contract.methods.installExtension(TEST_EXTENSION, 2).encodeABI();
+      const installExtension2Data = await colony.contract.methods.installExtension(TEST_VOTING_TOKEN, 1).encodeABI();
+      await checkErrorRevert(colony.multicall([installExtension1Data, installExtension2Data], { from: USER }), "ds-auth-unauthorized");
+
+      let extension1Address = await colonyNetwork.getExtensionInstallation(TEST_EXTENSION, colony.address);
+      let extension2Address = await colonyNetwork.getExtensionInstallation(TEST_VOTING_TOKEN, colony.address);
+
+      expect(extension1Address).to.equal(ethers.constants.AddressZero);
+      expect(extension2Address).to.equal(ethers.constants.AddressZero);
```

Comment:
I don't know if it's necessary to test the zero address here since we've already established the transaction failed, it just makes the tests longer and harder to parse.

Reply:
Have we established that the transaction failed? What if, to pick a totally random example, someone refactored `checkErrorRevert` incorrectly :laughing:

Reply:
I suppose so, especially in light of that. It's never wrong to be rigorous.